src/nxt_otel.h

```diff
@@ -11,13 +11,23 @@
 
 // forward declared
 struct nxt_http_field_t;
+struct nxt_conf_validation_t;
+struct nxt_conf_value_t;
 
-extern void   nxt_otel_send_trace(void *);
-extern void * nxt_otel_get_or_create_trace(u_char *);
-extern void   nxt_otel_init(void (*)(u_char*));
-extern void   nxt_otel_copy_traceparent(u_char *, void *);
-extern void   nxt_otel_add_event_to_trace(void *, u_char *, u_char *);
-extern void   nxt_otel_end_span(void *);
+extern void     nxt_otel_send_trace(void *);
+extern void   * nxt_otel_get_or_create_trace(u_char *);
+extern void     nxt_otel_init(void (*)(u_char*), const char *, const char *, double);
+extern void     nxt_otel_copy_traceparent(u_char *, void *);
+extern void     nxt_otel_add_event_to_trace(void *, u_char *, u_char *);
```

Comment:
This just seems like a lot of unnecessary churn. Why not just do it like this in the first place?